src/java.base/share/classes/jdk/internal/reflect/Reflection.java

```diff
@@ -53,6 +53,9 @@ public class Reflection {
     public static final Set<String> ALL_MEMBERS = Set.of(WILDCARD);
 
     static {
+        // 3 filter scenarios:
```

Comment:
Using "3" is a bit confusing here because the next line starts with "1.". So maybe change it to "Three" or drop completely as I don't think they are too helpful.